### PR TITLE
ENH: Pass scan kws to run start document

### PR DESCRIPTION
**Pull request by ericdill**

### CODE REVIEW

File: ophyd/userapi/scan_api.py [hunk 1]
```diff
@@ -17,7 +17,7 @@
 session_manager = get_session_manager()
 logger = session_manager._logger
 
-__all__ = ['AScan', 'DScan', 'Scan', 'Data', 'Count']
+__all__ = ['AScan', 'DScan', 'Data', 'Count']
```
Comment (tacaswell) [inline ophyd/userapi/scan_api.py]: Was this change intentional?

Comment (ericdill) [inline ophyd/userapi/scan_api.py]: Yes.

Comment (ericdill) [inline ophyd/userapi/scan_api.py]: sitting with @dchabot and @danielballan at XPD. dan and daron discussed it and I simply added the change to this PR

Comment (tacaswell) [inline ophyd/userapi/scan_api.py]: Can you add a note about the API change?

Comment (stuwilkins) [inline ophyd/userapi/scan_api.py]: Why are we making this change?

Best wishes,
Stuart

**Comment by danielballan** [comment]
@stuwilkins Scan does not have interaction with Olog, and the only thing is does with trajectories is check that they are within the bounds of the positions. Kevin used it by mistake, and obviously wasted some time, so @dchabot, @ericdill, and I think it muddies the waters. But if it is serving a purpose we don't know about, it should stay.

@tacaswell Yes.
